pkgs/applications/video/hdhomerun-config-gui/default.nix

```diff
@@ -1,12 +1,15 @@
-{ lib, stdenv, fetchurl, libhdhomerun, gcc, gnumake, pkg-config, gtk2 }:
+{ lib, stdenv, fetchurl, libhdhomerun, pkg-config, gtk2 }:
 
 stdenv.mkDerivation rec {
+
+# Version of libhdhomerun need to match the version of this package for successful build
+
   pname = "hdhomerun-config-gui";
-  version = "20200907";
+  version = "20210224";
```

Comment:
```suggestion
  # Version of libhdhomerun need to match the version of this package for successful build
  version = "20210224";
```

Reply:
I tried to commit this change but then the build fails.

Reply:
There is no reason the build should fail. Adding a comment is a noop.